File: src/main/java/no/nav/veilarbaktivitet/oppfolging/client/OppfolgingPeriodeMinimalDTO.java
package no.nav.veilarbaktivitet.oppfolging.client;


import java.time.ZonedDateTime;
import java.util.UUID;

public record OppfolgingPeriodeMinimalDTO(
    UUID uuid,
    ZonedDateTime startDato,
    ZonedDateTime sluttDato)
{
}

File: src/main/kotlin/no/nav/veilarbaktivitet/oppfolging/OppfolgingPeriodeMinimalDTO.kt
package no.nav.veilarbaktivitet.oppfolging

import java.time.ZonedDateTime
import java.util.*

data class OppfolgingPeriodeMinimalDTO(
    val uuid: UUID,
    val startDato: ZonedDateTime,
    val sluttDato: ZonedDateTime?
)
